openvdb/points/PointCount.h

```diff
@@ -117,6 +117,36 @@ Index64 inactiveGroupPointCount(const PointDataTreeT& tree, const Name& name,
     const bool inCoreOnly = false);
 
 
+/// @brief Total points in a tree which are included by a filter
+/// @param tree PointDataTree.
+/// @param filter Filter of the specified template type
+/// @param inCoreOnly if true, points in out-of-core leaf nodes are not counted
+template <typename PointDataTreeT, typename FilterT>
+Index64 filterPointCount(const PointDataTreeT& tree,
```

Comment:
A lot of this functionality was written before we went over to C++11, so features like default function template arguments weren't available then! Having access to this kind of functionality could improve this public API, I think perhaps something like this would be cleaner than having separate pointCount() and filterPointCount() functions:

```
template <typename PointDataTreeT, typename FilterT = NullFilter>
Index64 pointCount(...)
```

What do you think?